sdk/go/client.go

```diff
@@ -72,7 +72,13 @@ func WithLogOutput(writer io.Writer) ClientOpt {
 }
 
 // Connect to a Dagger Engine
-func Connect(ctx context.Context, opts ...ClientOpt) (*Client, error) {
+func Connect(ctx context.Context, opts ...ClientOpt) (_ *Client, rerr error) {
+	defer func() {
+		if rerr != nil {
+			rerr = withErrorHelp(rerr)
```

Comment:
Will this actually wrap the error in `return nil, err`? I thought everywhere had to assign `rerr` :thinking:

Reply:
:exploding_head: neat